src/block/component/CollisionBoxComponent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace customiesdevs\customies\block\component;
+
+use pocketmine\math\Vector3;
+use pocketmine\nbt\tag\CompoundTag;
+use pocketmine\nbt\tag\FloatTag;
+use pocketmine\nbt\tag\ListTag;
+
+class CollisionBoxComponent implements BlockComponent {
+
+	private bool $useCollisionBox;
+	private Vector3 $origin;
+	private Vector3 $size;
+
+	/**
+	 * Defines the area of the block that collides with entities. If set to true, default values are used. If set to false, the block's collision with entities is disabled. If this component is omitted
+	 * @param Vector3 $origin Minimal position of the bounds of the collision box. "origin" is specified as [x, y, z] and must be in the range (-8, 0, -8) to (8, 16, 8), inclusive.
+	 * @param Vector3 $size Size of each side of the collision box. Size is specified as [x, y, z]. "origin" + "size" must be in the range (-8, 0, -8) to (8, 16, 8), inclusive.
+	 * @param bool $useCollisionBox If Collision Should be Enabled, Default is set to `true`
```

Comment:
Would just move this line above the other `@param` docs to match the other, there's also a lot of capitals that are not needed
```diff
- * @param bool $useCollisionBox If Collision Should be Enabled, Default is set to `true`
+ * @param bool $useCollisionBox If collisions should be enabled, default is set to `true`.
```

Reply:
i copied pasted it from wiki